v2/emailpassword/custom-ui/init/dashboard.mdx

```diff
@@ -140,39 +140,9 @@ To update credentials you need to make a request to SuperTokens core.
 - Replace `<YOUR-API-KEY>` with your API key. If you are using a self hosted SuperTokens core there is no API key by default. In that case you can either skip or ignore the `api-key` header.
```

Comment:
the images in https://deploy-preview-752--admiring-bhabha-7b1be9.netlify.app/docs/thirdpartyemailpassword/pre-built-ui/setup/user-management-dashboard/users-listing-and-details are VERY VERY large.. way larger than what we are rendering. Please downsize.